app/styleguide/routes/links.vue

```diff
@@ -0,0 +1,35 @@
+<template>
```

Comment:
- [ ] Please move this example to the styleguide (typography) and remove this page.